src/app/docs/id/id-docs.component.html

```diff
@@ -100,8 +90,10 @@
       >
         <p>
           Don't use the ID directive when an element does not need to be referenced by its
-          ID attribute, since doing so adds unnecessary overhead to an application. The same rule
-          goes for assigning a hard-coded ID to an element, because it can lead to ID collisions.
+          ID attribute.
```

Comment:
If we keep `id` be as lowercase code snippet in the previous item, I'd change it here to match.